pyproject.toml

```diff
@@ -162,8 +162,8 @@ addopts = [
 
 # configuring https://github.com/pydantic/hooky
 [tool.hooky]
-assignees = ['samuelcolvin', 'adriangb', 'dmontagu', 'hramezani', 'lig', 'Kludex', 'davidhewitt']
-reviewers = ['samuelcolvin', 'adriangb', 'dmontagu', 'hramezani', 'lig', 'Kludex', 'davidhewitt']
+assignees = ['sydney-runkle']
```

Comment:
I think remove this or make it empty, doesn't really help if you're assigned to every new issue.